### PR TITLE
Implementing Wallet.import functionality

### PR DESCRIPTION
**Pull request by erdimaden**
### What changed? Why?
Moving import functionality to under Wallet Class

### CODE REVIEW

File: src/coinbase/wallet.ts [hunk 1]
```diff
@@ -89,6 +89,28 @@ export class Wallet {
     return Wallet.init(response.data!, "");
   }
 
+  /**
+   * Imports a Wallet for the given Wallet data.
+   *
+   * @param data - The Wallet data to import.
```
Comment (John-peterson-coinbase) [inline src/coinbase/wallet.ts:95]: nit: add params for `data.walletId` and `data.seed`. Also in doc string for user class `importWallet`

File: src/coinbase/wallet.ts [hunk 2]
```diff
@@ -89,6 +89,28 @@ export class Wallet {
...
+  public static async import(data: WalletData): Promise<Wallet> {
+    if (!data.walletId) {
+      throw new ArgumentError("Wallet ID must be provided");
+    }
+    if (!data.seed) {
+      throw new ArgumentError("Seed must be provided");
+    }
+    const walletModel = await Coinbase.apiClients.wallet!.getWallet(data.walletId);
+    const wallet = Wallet.init(walletModel.data, data.seed);
+    await wallet.listAddresses();
```
Comment (John-peterson-coinbase) [inline src/coinbase/wallet.ts:110]: why are we listing addresses here?

Comment (jazz-cb) [inline src/coinbase/wallet.ts:110]: we dont seem to be listing addresses here https://github.com/coinbase/coinbase-sdk-ruby/blob/master/lib/coinbase/wallet.rb#L38

Comment (erdimaden) [reply src/coinbase/wallet.ts:110]: Good catch! it was pre-fetching addresses, but it is no longer necessary here. removing it.

**Review by jazz-cb — APPROVED** [review]

**Pull request merged**